pyaedt/icepak.py

```diff
@@ -4194,39 +4140,30 @@ def assign_solid_block(
         ----------
         object_name : str or list
             Object name or a list of object names.
-        power_assignment : str or dict
+        power_assignment : str or dict or BoundaryDictionary
             String with the value and units of the power assignment or with
-            ``"Joule Heating"``. For a temperature-dependent or transient
-            assignment, a dictionary can be used. The dictionary should contain three keys:
-            ``"Type"``, ``"Function"``, and ``"Values"``.
-            - For the ``"Type"`` key, accepted values are ``"Temp Dep"`` and ``"Transient"``.
-            - For the ``"Function"`` key, acceptable values depend on the ``"Type"`` key
-            selection. When the ``"Type"`` key is set to ``"Temp Dep"``, the only
-            accepted value is ``"Piecewise Linear"``. When the ``"Type"`` key is
-            set to ``"Transient"``, acceptable values are `"Exponential"``, `"Linear"``,
-            ``"Piecewise Linear"``, ``"Power Law"``, ``"Sinusoidal"``, and ``"SquareWave"``.
-            - For the ``"Values"`` key, a list of strings contain the parameters required by
-            the ``"Function"`` key selection. For example, when``"Linear"`` is set as the
-            ``"Function"`` key, two parameters are required: the value of the variable
-            at t=0 and the slope of the line. For the parameters required by each
-            ``"Function"`` key selection, see the Icepak documentation. The parameters
-            must contain the units where needed.
+            ``"Joule Heating"``.
```

Comment:
The carryon of the sentence ("or with " ``"Joule Heating"`` doesn't seem clear to me.

Reply:
It means that if you don't want to assign a specific power but set a joule heating dissipation, you can just use power_assignment="Joule Heating".

Do you have any suggestion to make it clearer?

Reply:
LOL! I used exactly what you said to explain it! That made everything quite clear!